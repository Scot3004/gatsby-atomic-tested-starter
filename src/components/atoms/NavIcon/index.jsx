import styled from 'styled-components';

export const NavIcon = styled.span``;

export default NavIcon;
