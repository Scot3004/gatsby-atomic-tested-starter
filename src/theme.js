const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
