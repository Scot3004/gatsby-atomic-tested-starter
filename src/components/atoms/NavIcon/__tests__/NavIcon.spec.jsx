import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import NavIcon from '..';

describe('When I see an icon', () => {
  const tree = renderer.create(<NavIcon theme={theme} />).toJSON();
  test('Then it should be a inline element', () => {
    expect(tree).toMatchSnapshot();
  });
});
