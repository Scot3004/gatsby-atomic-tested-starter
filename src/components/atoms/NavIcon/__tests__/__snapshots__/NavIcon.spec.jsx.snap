// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I see an icon Then it should be a inline element 1`] = `
<span
  className=""
/>
`;
